### DISCUSSION TITLE
How can I use CoreForms? and How can I add a custom form for a survey?

### DISCUSSION THREAD

**Discussion by Swrnv-qc** [wrappid/.github#19] [Q&A]
I want to make a form for a simple survey like google forms so a step by step guide would be very helpful.

**Comment by samhere17** [comment]
Presently we do not have a form builder like Google form. So, as of now, you will have to create and pass json as argument to CoreForm.

@sumanta-m @techoneel may be able to help you. They are the primary designers of CoreForm.

**Comment by techoneel** [comment]
We can achieve this by following steps:

1. Project Creation - Create an app and a service project using toolkit.
2. Project Setup
    1. App - Add a CoreForm component in your default route page component and provide schema through formID or formJson props by following documentation available [here](//demo.wrappid.dev/guide/components)
    2. Service - Create a database table and model schema to store your survey form data and use the /data/:model_name API to post data from your form.
3. Project Start - Start your app and service project to see it's working

@Swrnv-qc closing this discussion as of now and you can reopen if you want more help with the process or have any suggestion.
